monitoring/ping.go

```diff
@@ -160,7 +160,7 @@ func (c *pingChecker) checkNodesRTT(nodes []serf.Member, client *serf.RPCClient)
 		}
 
 		rttStatsHDRInterface, _ := c.rttStats.Get(node.Name)
-		rttStatsHDR := rttStatsHDRInterface.(hdrhistogram.Histogram)
+		rttStatsHDR := rttStatsHDRInterface.(*hdrhistogram.Histogram)
```

Comment:
this should not panic, so should be `rttStatsHDR, ok :=` and next line `if !ok {return err}`.

Also `rttStatsHDR` was hard for me to read, can we rename it to something like `roundtripLatency`. You don't need to communicate the type of the variable, because Go does this automatically - it's known that it's a Histogram.